### PR TITLE
Moved arithematic operators to sympy/sets/sets.py

### LINKED ISSUE #14366: Move set arithmetic operators to sympy/sets/sets.py
Set arithmetic operators such as `set_add` and the function application on sets (`set_function`) have been defined in `sympy/sets/setexpr.py` to support `SetExpr`.

They are actually of generic use for all sets, so they should rather stay in `sympy/sets/sets.py`.

### PR DESCRIPTION
**Pull request by rahilhastu**
All the functions related to generic use for all sets are moved from
sympy/sets/setexpr.py to sympy/sets/sets.py

#### References to other Issues or PRs

Fixes #14366

### CODE REVIEW

File: sympy/sets/sets.py [hunk 1]
```diff
@@ -1959,3 +1957,33 @@ def simplify_intersection(args):
...
+    return _apply_operation(_set_div, x, y, commutative=False)
+
+def set_pow(x, y):
+    from sympy.sets.handlers.power import _set_pow
+    from sympy.sets.setexpr import _apply_operation
+    return _apply_operation(_set_pow, x, y, commutative=False)
+
+def set_function(f, x):
+    from sympy.sets.handlers.functions import _set_function
+    from sympy.sets.setexpr import _apply_operation
```
Comment (lilithxxx) [inline sympy/sets/sets.py]: ```_apply_operation``` function is not needed here so no need to import it here.

Comment (rahilhastu) [reply sympy/sets/sets.py]: > _apply_operation function is not needed here so no need to import it here.

removed it. 
Is it fine now?

File: sympy/sets/sets.py [hunk 2]
```diff
@@ -1959,3 +1957,32 @@ def simplify_intersection(args):
         return args.pop()
     else:
         return Intersection(args, evaluate=False)
+
+def set_add(x, y):
+    from sympy.sets.handlers.add import _set_add
+    from sympy.sets.setexpr import _apply_operation
```
Comment (Upabjojr) [inline sympy/sets/sets.py]: `_apply_operation` and `_handle_finite_sets` should also be moved, in my opinion.

Comment (rahilhastu) [reply sympy/sets/sets.py]: do I need to move it? @lilithxxx

**Comment by Upabjojr** [comment]
Thanks for contributing!

**Pull request merged**